sqlc.yaml

```diff
@@ -178,6 +180,39 @@ sql:
       go:
         <<: *gengo
         out: "backend/controller/encryption/dal/internal/sql"
+  - <<: *daldir
+    queries:
+      - backend/controller/timeline/dal/internal/sql/queries.sql
+      - backend/controller/timeline/dal/internal/sql/deployment_queries.sql
+    gen:
+      go:
+        <<: *gengo
+        out: "backend/controller/timeline/dal/internal/sql"
+        overrides:
+          - db_type: "encrypted_timeline"
+            go_type: "github.com/TBD54566975/ftl/internal/encryption.EncryptedTimelineColumn"
+          - db_type: "encrypted_timeline"
+            nullable: true
+            go_type: "github.com/TBD54566975/ftl/internal/encryption.OptionalEncryptedTimelineColumn"
+          - db_type: "timestamptz"
+            go_type: "time.Time"
+          - db_type: "timestamptz"
+            nullable: true
+            go_type:
+              type: "optional.Option[time.Time]"
+          - db_type: "pg_catalog.interval"
+            go_type: "github.com/TBD54566975/ftl/backend/controller/sql/sqltypes.Duration"
+          - db_type: "pg_catalog.interval"
```

Comment:
We don't want to duplicate these types, just keep all the type mappings in the root overrides.

Reply:
Hmm I must be doing something weird because if I don't have these it seems like the code won't build anymore. Seems like stuff generates like this:

```go
type InsertTimelineCallEventParams struct {
	DeploymentKey    interface{}
	RequestKey       sql.NullString
	ParentRequestKey sql.NullString
	TimeStamp        time.Time
	SourceModule     sql.NullString
	SourceVerb       sql.NullString
	DestModule       string
	DestVerb         string
	Payload          encryption.EncryptedTimelineColumn
}
```

Instead of `optional.Option[string]` for example. Maybe I need another change for the `sqlc` config

Reply:
I dunno. I deleted it, merged it, and everything compiled fine... 🤷‍♂️